### PR TITLE
Allow on-ride photo segments on Bobsleighs

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/sprites.h [hunk 1]
```diff
@@ -671,6 +671,9 @@ enum {
     SPR_RIDE_DESIGN_PREVIEW_MAZE_ICE_BLOCKS = 21992,
     SPR_RIDE_DESIGN_PREVIEW_MAZE_WOODEN_FENCES = 21993,
 
+    // Used by on-ride photo segment and station
+    SPR_METAL_PLATE = 22432,
```
Comment (marijnvdwerf) [inline src/openrct2/sprites.h]: Also known as `SPR_STATION_BASE_D`

Comment (Gymnasiast) [inline src/openrct2/sprites.h]: Yes, I saw that name, but I considered it overly specific.

@IntelOrca ?

Comment (IntelOrca) [inline src/openrct2/sprites.h]: I don't know, do we really need two constants for the same thing?

Comment (Gymnasiast) [inline src/openrct2/sprites.h]: No, but we need to pick a name. Either `SPR_STATION_BASE_D` or `SPR_METAL_PLATE` or something more descriptive.

Comment (IntelOrca) [inline src/openrct2/sprites.h]: `SPR_STATION_BASE_METAL_PLATE`?

Comment (marijnvdwerf) [inline src/openrct2/sprites.h]: image
There are 4 sets(SE/SW), with differing "pole" heights.

**Pull request merged**